compose/config/config.py

```diff
@@ -76,6 +76,7 @@
     'fig.yaml',
 ]
 
+DEFAULT_OVERRIDE_FILENAME = 'docker-compose.override.yml'
```

Comment:
Other defaults we've put in `const.py`, does this belong in there with them as well?